src/context/index.tsx

```diff
@@ -33,6 +33,10 @@ const ContextProvider = (props: PropsWithChildren) => {
           name: 'customConfig',
           apiTimeout: 6000,
           walletConnectV2ProjectId: '9b1a9564f91cb659ffe21b73d5c4e2d8',
+          metamaskSnapWalletAddress:
+            state.dappEnvironment === EnvironmentsEnum.mainnet
+              ? 'https://snap-wallet.multiversx.com'
```

Comment:
what about testnet?

Reply:
i removed it, radu added defaults in skd-dapp